google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineDeploymentConfiguration.java

```diff
@@ -49,6 +49,7 @@
 
   public static final String USER_SPECIFIED_ARTIFACT_PATH_ATTRIBUTE = "userSpecifiedArtifactPath";
   public static final String STAGED_ARTIFACT_NAME = "stagedArtifactName";
+  public static final String STAGED_ARTIFACT_NAME_LEGACY = "stagedArtifactNameLegacy";
```

Comment:
Didn't we say we wouldn't call this "legacy"? I think this was @patflynn's suggestion

Reply:
IIRC, @patflynn's comment was to not simply call it "legacy" because it's too obscure and we may have other "legacy" things to migrate in the future.